mill

```diff
@@ -53,7 +53,20 @@ elif [[ "$cache_dest" == *.zip ]]; then
   fi
 fi
 
-eval "$("$cs" java --env --jvm temurin:17 || "$cs" java --env --jvm openjdk:1.17.0)"
+function fixit {
```

Comment:
maybe a more informative name for this function would be better?
```suggestion
function adapt_for_windows {
```
As far as I understand, this is to fix this script from Unix-style to  Windows-style.

Reply:
The purpose is to fix incorrect syntax returned by `cs` (a fix has been submitted to coursier).  So maybe it could be renamed as `to_bash_syntax()`